src/_includes/section-menu/contribute.html

```diff
@@ -16,6 +16,7 @@
 
   <ul class="section-nav-list">
     <li><a href="{{ site.baseurl }}/contribute/design-principles/">Design principles</a></li>
+    <li><a href="{{ site.baseurl }}/contribute/portability/">Portability Framework</a></li>
```

Comment:
will need to remove this entry as a workaround for the issue when testing for new files, and put in a separate PR after this one is in. there's a JIRA for me to fix the root cause of that (BEAM-3144)

Reply:
Sorry, but are you asking me to do that or are you doing it?

Reply:
Done